core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -517,8 +394,8 @@
      * trace while the others will have an empty stack trace to eliminate the cost of capturing the stack
      * trace.
      *
-     * <p>The default value of this flag is {@value #DEFAULT_VERBOSE_EXCEPTION_SAMPLER_SPEC}, which retains
-     * the stack trace of the exceptions at the maximum rate of 10 exceptions/sec.
+     * <p>The default value of this flag is {@link DefaultFlagsProvider#verboseExceptionSamplerSpec}, which
```

Comment:
We cannot link an internal API to public Javadoc. Hmm. Constant values are needed to define in `DefaultFlagsProvider` for Javadoc?

Reply:
I got your point.

> Constant values are needed to define in DefaultFlagsProvider for Javadoc?

Do you mean like this?

 ```java
     *
     * <p>The default value of this flag is {@value DefaultFlagsProvider#defaultVerboseExceptionSamplerSpec}, which
     * retains the stack trace of the exceptions at the maximum rate of 10 exceptions/sec.
```

```java
// constant need to be package-private so that Flags can access it
static final String defaultVerboseExceptionSamplerSpec = "rate-limit=10";
    @Override
    public String verboseExceptionSamplerSpec() {
        return defaultVerboseExceptionSamplerSpec;
    }
```

Reply:
> Do you mean like this?

Correct.